plugin/DataView/DataDetailViewHandler.php

```diff
@@ -67,7 +68,15 @@ public function getDetailView(): DataDetailView
 		{
 			$pResult = $pAlternate;
 		}
-
+        if(empty($pResult->getPageId()))
+        {
+            $pRecordPostMeta = new RecordManagerPostMeta();
```

Comment:
`RecordManagerPostMeta` instance should be obtained in constructor.

Reply:
That way, it could be mocked in the test class of DataDetailViewHandler, if needed